src/main/java/org/truffleruby/core/string/StringSupport.java

```diff
@@ -79,6 +79,11 @@ public static boolean MBCLEN_NEEDMORE_P(int r) {
         return r < -1;
     }
 
+    // MBCLEN_NEEDMORE_LEN, ONIGENC_MBCLEN_NEEDMORE_LEN
+    public static int MBCLEN_NEEDMORE_LEN(int r) {
+        return -1-r;
```

Comment:
yeah, I've just tracked down https://github.com/ruby/ruby/blob/v2_3_3/include/ruby/oniguruma.h#L254 and reimplemented.